functions/colab.js

```diff
@@ -48,7 +48,14 @@ exports.handler = async function (event, _) {
     '!pip install -r requirements.txt'
   ]
 
-  const execution_nb_commands = ['!python main.py config.yaml']
+  let argparser = data.argparser
+  let execution_nb_commands = []
+
+  if (argparser == 'hydra') {
+    specific_commands.push('!python main.py')
```

Comment:
Is there a way to specify the config file explicitly ? It would be better than implicit one

Reply:
Yeah you can say `python main.py --config-dir=[dir-path] --config-name=[config-name] ` as given in Hydra docs \